src/swtpm/capabilities.c

```diff
@@ -43,6 +43,7 @@
 #include <string.h>
 
 #include <libtpms/tpm_library.h>
+#include <libtpms/tpm_error.h>
```

Comment:
Can you prefix the title of this commit with `swtpm:`

Reply:
sure

Reply:
Can you please prefix the title of this patch with `swtpm:`.